cluster-state-service/swagger/v1/swagger.json

```diff
@@ -248,8 +264,30 @@
     }
   },
   "definitions": {
+    "Metadata": {
+      "type": "object",
+      "required": [
+        "entityVersion"
+      ],
+      "properties": {
+        "entityVersion": {
+          "type": "string"
+        }
+      }
+    },
     "ContainerInstance": {
       "type": "object",
+      "properties": {
+        "metadata": {
+          "$ref": "#/definitions/Metadata"
```

Comment:
Ok, keeping single Metadata object for now. Should be an easy change in the future if we want.